packages/@aws-cdk/aws-servicecatalog/lib/portfolio.ts

```diff
@@ -100,6 +100,14 @@ export interface IPortfolio extends cdk.IResource {
    */
   notifyOnStackEvents(product: IProduct, topic: sns.ITopic, options?: CommonConstraintOptions): void;
 
+  /**
+   * Set provisioning rules for the product.
+   * @param product A service catalog product.
+   * @param assertion A list of valid rules to apply.
+   * @param options options for the constraint.
+   */
+  constrainProvisioningParameters(product:IProduct, assertion: TemplateRule, options?: CommonConstraintOptions): void;
```

Comment:
Instead of this method having 3 parameters, create a new interface that extends `CommonConstraintOptions`, has the `readonly assertion: TemplateRule` property, and make it the second argument of this method.

Reply:
sure